app/dashboard/src/components/AriaComponents/Form/components/Submit.tsx

```diff
@@ -26,7 +26,7 @@ interface SubmitButtonBaseProps {
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
   readonly form?: FormInstance<any>
   /** Defaults to `submit`. */
-  readonly action?: 'cancel' | 'submit' | 'update'
+  readonly action?: 'cancel' | 'save' | 'submit' | 'update'
```

Comment:
how's the save is different from 'submit' or 'update'? 

On the second thought, why'd we need that at all? we had `formnovalidate` which was _quirky_, but web-compliant, now we have custom prop, that looks very unclear and has very confusing values.